ethers-providers/Cargo.toml

```diff
@@ -24,6 +24,7 @@ serde_json = { version = "1.0.64", default-features = false }
 thiserror = { version = "1.0.24", default-features = false }
 url = { version = "2.2.1", default-features = false }
 auto_impl = { version = "0.4.1", default-features = false }
+futures = { version = "0.3.5", default-features = false, optional = true }
```

Comment:
You do not need this dep, you can get `StremExt` from `futures-util` https://docs.rs/futures-util/0.3.13/futures_util/stream/trait.StreamExt.html